lib/modules/apostrophe-browser-utils/public/js/always.js

```diff
@@ -62,15 +62,19 @@ apos.define('apostrophe-browser-utils', {
       var i;
       var css = '';
       var dash = false;
+      var start = true;
       for (i = 0; (i < camel.length); i++) {
         var c = camel.charAt(i);
         var lower = ((c >= 'a') && (c <= 'z'));
         var upper = ((c >= 'A') && (c <= 'Z'));
         var digit = ((c >= '0') && (c <= '9'));
         if (!(lower || upper || digit)) {
-          dash = true;
+          if (!start) {
```

Comment:
Can't we simplify this by checking if `i === 0`, removing the `start` variable?

Reply:
That would not work if the string started with several non-alpha-digit characters that should not turn into leading `-`'s.